src/Fantomas.Tests/IfThenElseTests.fs

```diff
@@ -1651,14 +1651,14 @@ let tryDecompile (ty: FSharpEntity) =
     match ty.TryFullName with
     | Some fullName -> return decompile ty.Assembly.SimpleName externalSym
     | None ->
-        // might be abbreviated type (like string)
```

Comment:
Just checking that in the final version, you'll also keep the tests for when the feature is turned off? (I'm happy with this test case though.)